packages/chat-core-aws-connect/test-browser-esm/rollup.config.js

```diff
@@ -0,0 +1,27 @@
+import resolve from "@rollup/plugin-node-resolve";
```

Comment:
thoughts on pulling test-browser-esm and test-node-cjs to a top level `test-sites/`? So chat-core and chat-core-aws-connect can both share the setup. We can have different pages or buttons to test each of of the "core" library

Reply:
I like that idea! Do you mind if we make that a followup item? Might be a good thing for an intern to pick up 👀

Reply:
https://yexttest.atlassian.net/browse/CLIP-1338